torchani/data/data.py

```diff
@@ -357,10 +133,28 @@ def release_h5(self):
         gc.collect()
 
 
-def ShuffleDataset(file_path, batch_size=1000, bar=20, test_bar_max=None, num_workers=0, shuffle=True):
-    dataset = TorchData(file_path)
+def ShuffleDataset(file_path,
+                   species_order='HCNO',
+                   transform=True,
```

Comment:
@zasdfgbnm , what about this?
if user want transform,
he needs to set `transform=True`
and provide a `self_energies` list